src/declarations/stencil-private.ts

```diff
@@ -32,6 +34,16 @@ import type {
   VNodeData,
 } from './stencil-public-runtime';
 
+export interface TerserSourceMap {
+  mappings: SourceMapMappings;
+  file?: string;
+  names: string[];
+  sourceRoot?: string;
+  sources: string[];
+  sourcesContent?: string[];
+  version: 3;
+}
+
 export interface SourceMap {
```

Comment:
I wonder if we need this interface still 🤔 

I'll take a closer look at that in a bit.